execution/block-partitioner/src/v2/load_balance.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright © Aptos Foundation
+
+use itertools::Itertools;
+use std::collections::BinaryHeap;
+
+/// The longest-processing-time-first scheduling.
+///
+/// Time complexity: O(num_tasks * log2(num_workers))
+pub fn longest_processing_time_first(tasks: &Vec<u64>, num_workers: usize) -> (u64, Vec<usize>) {
```

Comment:
Perhaps a better name to first param would be 'task_costs' or 'task_to_cost' ?

Reply:
fixed